kuma/javascript/src/breadcrumbs.jsx

```diff
@@ -8,6 +11,22 @@ type DocumentProps = {
 };
 
 export default function Breadcrumbs({ document }: DocumentProps) {
+    const ga = useContext(GAProvider.context);
+
+    /**
+     * Send a signal to GA when there is an interaction on one
+     * of the breadcrumb menu links.
+     * @param {String} label  - The label for the GA event
+     */
+    function sendBreadcrumbItemClick(label) {
```

Comment:
Can this also please take `event` instead and on line 26 do `event.target.href` instead? That way you're not creating new functions on ever render.